doc/encoders.texi

```diff
@@ -3564,6 +3564,22 @@ bitrate, @var{target_bitrate}, within the accuracy range @var{avbr_accuracy},
 after a @var{avbr_Convergence} period. This method does not follow HRD and the
 instant bitrate is not capped or padded.
 
+@item @var{skip_frame}
+Use per-frame metadata "qsv_skip_frame" to skip frame when encoding. This option
+defines the usage of this metadata.
+@table @samp
+@item no_skip
+Frame skipping is disabled.
+@item insert_dummy
+Encoder inserts into bitstream frame where all macroblocks are encoded as
+skipped.
+@item insert_nothing
+Similar to insert_dummy, but encoder inserts nothing into bitstream.
```

Comment:
So the current frame is skipped in the stream, right ?

Reply:
Yes.

Reply:
If so, why does user application not encode the current frame ?

Reply:
The skipped frames will still be used for brc. I added more explanation into doc.